Console/Decoify/template/main-header.php

```diff
@@ -15,6 +156,38 @@
 
       <div class="navbar-custom-menu">
         <ul class="nav navbar-nav">
+<?php
+if(isset($_SESSION['user_name']) && $_SESSION['role'] == 'admin'){ 
```

Comment:
This should be beginning of the code to avoid processing for unauthenticated user.

Reply:
Previously this condition was not in main-header.php so now I put this condition above the php code of health check code right? In other links except health check this condition should not be there as it is before?